modules/smart-agent_kubernetes-workloads-count/conf/01-kubernetes-workloads-count.yaml

```diff
@@ -0,0 +1,24 @@
+module: "Kubernetes"
+name: "Workloads Count"
+filtering: "not filter('kubernetes_namespace','ara') and not filter('kubernetes_namespace','bastions') and not filter('kubernetes_namespace','gitlab-runner') and not filter('kubernetes_namespace','logging') and not filter('kubernetes_namespace','monitoring') and not filter('kubernetes_namespace','ingress-nginx') and not filter('kubernetes_namespace','kube-system') and not filter('kubernetes_namespace','kubernetes-replicator')"
```

Comment:
this comment is optional and mainly for the record.

sadly we cannot do better using the yaml/generator config.. but we could remove the yaml conf to commit only the terraform code as "handmade" detector.

From this way you could use a terraform variable for namespaces exclusion, this will allow users to easily change or add namespaces to the default ones without to fork / PR in future.